ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/pderesources.properties

```
@@ -488,6 +488,7 @@ SchemaAttributeDetails_use=Use:
 AbstractPluginBlock_counter={0} out of {1} selected
 AbstractTargetPage_setTarget=Set as Active Target Platform
 AbstractTargetPage_reloadTarget=Reload Target Platform
+AbstractTargetPage_showAll=Show all Target Platforms
```

Although it is true that the opened preference page shows all target-definitions, what the action is actually doing is opening the `Target Platform` preference page.
Therefore I think the tool-tip should be accordingly:
```suggestion
AbstractTargetPage_showAll=Open 'Target Platform' preference
```
If you want to also hint that it shows all targets, you could append something like `(lists all available targets)`